BiweeklyMeetings/2022-07-05.md

```diff
@@ -0,0 +1,274 @@
+**Sebastien** -:  Yeah. Makes sense to me.
+
+**Matthias** - :  And we can maybe do a bit of PR management at the same time and see if we can close the first one. I will just comment on this directly. It's marked as legacy deprecated already. Yeah. And Robert actually already commented on that. Okay, anything else? I see there is a small one, maybe PR number 275, sorry, which is just updating line numbers and this time, putting an actual permalink link in CIP 8, which I think is good. So initially, it was just updating line numbers, I think, but then Robert rightfully asked to just put a permalink link in there so that we don't get to update that line number every week. So 
+
+**Sebastien** -, that's probably something also you would have a minor comment on. I'm not sure if you are still involved with the CSL anymore, or if you are fully switched to the multi-platform library.
+
+**Sebastien** -:  Yeah. I haven't touched CSL for a long time. Yeah, for the comment related to WebPull, I don't know of a good alternative website that allows you to run WebAssembly in the browser for free. WebPull is the one of the few that did it, but I feel like now that WebAssembly is more common, there might be other platforms that work better.
+
+**Matthias** - :  Because a rep needs to be paid by an owner?
+
+**Sebastien** -:  Yeah. So it used to be free and then they changed it to paid, which is why it all links up probably.
+
+**Matthias** - :  Okay. So at the same time, it's also filled to the current PR, so perhaps that's one good use of an issue. That's the other rep link, it doesn't work anymore, fine. But at least this PR fixes already a dead link, which is good in itself, which is comment to that actually. Okay. Nope, doesn't work. So I will open an issue following that. And where is actually the comment about rep link? Here. Rep link eight. So rep eight link doesn't work more. Okay. So this one is done. Do we have any small PR like that or should we just wrap it up for today? Or anything else people want to bring forward? Nope? Oh sorry, I've got a phone call. Okay, sorry about that. 
+
+### Discussions
+N/A
+
+### Issues
+N/A
+
+### Close 
+
+So let's wrap it up, maybe?
+
+**Matthias** - :  So we went through CIP 59 today, the terminology that runs core features. We agreed to merge it as active, as it is informal CIP bringing clarity on naming and coming from the core team directly. Then we've looked at CIP 38, CIP 30, CIP 37, I think that's the only three ones, which are waiting for an update from the author. We discussed that for those CIP that has been idle and dormant for a few months, it might be good actually to just close the Pull Requests and well, let the author come back with either a new Pull Request or an update if they do. But before doing that or in parallel of doing that, also specify that, "In the CIP 1, as a small change or in the process," so that we are upfront with what's going to happen. And it'll help us keep the repository tidy.
```

Comment:
```suggestion
**Matthias** - :  So we went through CIP 59 today, the terminology that runs core features. We agreed to merge it as active, as it is informal CIP bringing clarity on naming and coming from the core team directly. Then we've looked at CIP 38, CIP 30, CIP 37, I think that's the only three ones, which are waiting for an update from their authors. We discussed that for those CIPs that have been idle and dormant for a few months, it might be good actually to just close the Pull Requests and well, let the authors come back with either a new Pull Request or an update if they do. But before doing that or in parallel of doing that, also specify that, "In the CIP 1, as a small change or in the process," so that we are upfront with what's going to happen. And it'll help us keep the repository tidy.
```